server/routes/creatingLicences/handlers/caseload.ts

```diff
@@ -35,17 +38,60 @@ export default class CaseloadRoutes {
       req.session.returnToCase = '/licence/create/caseload'
     }
 
-    const caseload = teamView
-      ? await this.comCaseloadService.getTeamCreateCaseload(user, req.session.teamSelection)
-      : await this.comCaseloadService.getStaffCreateCaseload(user)
+    const caseload = (
+      teamView
+        ? await this.comCaseloadService.getTeamCreateCaseload(user, req.session.teamSelection)
+        : await this.comCaseloadService.getStaffCreateCaseload(user)
+    ).map(comCase => {
+      return {
+        ...comCase,
+        createLink: this.findCreateLinkToDisplay(
+          comCase.licenceCreationType,
+          comCase.licenceId,
+          comCase.prisonerNumber
+        ),
+        releaseDate: format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy'),
+        hardStopDate: comCase.hardStopDate && format(parseCvlDate(comCase.hardStopDate), 'dd/MM/yyyy'),
+        hardStopWarningDate:
+          comCase.hardStopWarningDate && format(parseCvlDate(comCase.hardStopWarningDate), 'dd/MM/yyyy'),
+        isClickable:
+          comCase.probationPractitioner !== undefined &&
+          comCase.licenceStatus !== LicenceStatus.NOT_IN_PILOT &&
+          comCase.licenceStatus !== LicenceStatus.OOS_RECALL &&
```

Comment:
**nitpick:** No harm in the `OOS`s being here but I believe (and I may be getting confused as we've had various conversations about it) that we're removing `OOS_RECALL` and `OOS_BOTUS` as statuses, as we never actually use them for anything

Reply:
ok will remove